test/helper.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require "simplecov"
+require "simplecov_json_formatter"
+SimpleCov.formatter = SimpleCov::Formatter::JSONFormatter if ENV.fetch("CC_TEST_REPORTER_ID", false)
```

Comment:
These line shouldn't be necessary either, this behavior is already present on `simplecov`.

Reply:
Got it, thanks.